packages/cli/src/config/setup/Setup.handler.ts

```diff
@@ -0,0 +1,185 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import {
+    type ICommandHandler,
+    type IConfigLayer,
+    type IHandlerParameters,
+    type IHandlerResponseApi,
+    type IProfileTypeConfiguration,
+    ImperativeConfig,
+    ImperativeError,
+    ProfileInfo,
+    TextUtils,
+} from "@zowe/imperative";
+import * as termkit from "terminal-kit";
+import {
+    AbstractConfigManager,
+    MESSAGE_TYPE,
+    type ProgressCallback,
+    type inputBoxOpts,
+    type qpItem,
+    type qpOpts,
+} from "zowe-native-proto-sdk";
+
+export default class ServerInstallHandler implements ICommandHandler {
+    public async process(params: IHandlerParameters): Promise<void> {
+        const profInfo = new ProfileInfo("zowe");
+        await profInfo.readProfilesFromDisk();
+        const cliPromptApi = new CliPromptApi(profInfo, params.response);
+        const profile = await cliPromptApi.promptForProfile();
+        if (profile) {
+            params.response.console.log(
+                `SSH Profile Validated: ${(profInfo.getTeamConfig().api.layers.find(profile.name) as IConfigLayer).path}`,
+            );
+        }
+    }
+}
+
+export class CliPromptApi extends AbstractConfigManager {
+    constructor(
+        mProfilesCache: ProfileInfo,
+        private mResponseApi: IHandlerResponseApi,
+    ) {
+        super(mProfilesCache);
+    }
+
+    private term = termkit.terminal;
+    protected showMessage(message: string, type: MESSAGE_TYPE): void {
+        switch (type) {
+            case MESSAGE_TYPE.INFORMATION:
+                this.mResponseApi.console.log(message);
+                break;
+            case MESSAGE_TYPE.WARNING:
+                this.mResponseApi.console.log(TextUtils.chalk.yellow(message));
+                break;
+            case MESSAGE_TYPE.ERROR:
+                this.mResponseApi.console.log(TextUtils.chalk.red(message));
```

Comment:
Should we write error messages to `stderr`?
```suggestion
                this.mResponseApi.console.error(TextUtils.chalk.red(message));
```